vpr/src/route/rr_graph.cpp

```diff
@@ -2900,8 +2896,7 @@ static std::vector<bool> alloc_and_load_perturb_opins(const t_physical_tile_type
     return perturb_opins;
 }
 
-static RRNodeId pick_best_direct_connect_target_rr_node(const t_rr_graph_storage& rr_nodes,
-                                                        RRNodeId from_rr,
+static RRNodeId pick_best_direct_connect_target_rr_node(RRNodeId from_rr,
```

Comment:
Since this function has been refactored and the old data structure ``rr_nodes`` has been shadowed, I suggest to replace the ``const t_rr_graph_storage& rr_nodes`` with ``const RRGraphView& rr_graph`` as the first argument.